packages/ui-stencil/src/components/orama-chat-box/orama-chat-box.scss

```diff
@@ -34,3 +38,7 @@
     }
   }
 }
+
+.header.hidden {
```

Comment:
related to my comment above, I would avoid rendering this piece at all, so in this case we can get rid of this style